sdks/router-sdk/src/entities/mixedRoute/route.ts

```diff
@@ -13,9 +14,10 @@ type TPool = Pair | Pool
  */
 export class MixedRouteSDK<TInput extends Currency, TOutput extends Currency> {
   public readonly pools: TPool[]
-  public readonly path: Token[]
+  public readonly path: Currency[]
   public readonly input: TInput
   public readonly output: TOutput
+  public readonly adjustedInput: Currency // routes with v2/v3 initial pool must wrap native input currency before trading
```

Comment:
could find a better name for this... this input isn't necessarily the input from the user, but the "adjusted" input for trading on the pool. i.e. the user sends ETH, but trades on a WETH pool first (so it must be wrapped)

Reply:
ah yea, I don't mind the current naming, another idea could be `inputOrWrapped`?